jules-julesforrest/hw_2/5-random-address-generator-bonus/app.js

```diff
@@ -0,0 +1,23 @@
+/*
+Optional bonus work on conditionals and functions:
+
+    Write a script that can generate random addresses
+    As a first step, create arrays that contain dummy data for each of the following: street number, street name, city name, state name, and zip code
+    Your script should randomly select one item from each of these arrays and then use them to construct a random address
+    Each time you load or reload the page in a browser, your program should print a new randomly-generated address to the terminal. For example:
+
+    34578 Dolphin Street, Wonka NY, 44506
+*/
+
+var randElem = function(myArr){
+  return myArr[Math.floor(Math.random() * myArr.length)];
```

Comment:
Aha! Brilliant dynamic code you've written, Jules! It's easy to follow and understand this function's single responsibility, and again when it's used on arrays in the `addressGenerator()` function below.

You're going to have a blast when we get to module patterns, prototypal inheritance (classes in other languages), and async & callbacks. 😋